sktime/classification/dictionary_based/_cboss.py

```diff
@@ -267,7 +264,7 @@ def fit(self, X, y):
         self._is_fitted = True
```

Comment:
this line should also be removed

Reply:
Thanks for reviewing. I have made this change in new commit.

> this line should also be removed